lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -402,6 +402,26 @@ def __OnPrimsChanged(self, notice, sender):
                 if hasattr(ctrl, 'refresh'):
                     ctrl.refresh()
 
+def connectionsCustomControlCreator(self, c):
+    if self.attributeHasConnections(c):
+        connectionsCustomControl = ConnectionsCustomControl(self.item, self.prim, c, self.useNiceName)
+        self.defineCustom(connectionsCustomControl, c)
```

Comment:
Shouldn't this be done by AETemplate.addControls() code?  Otherwise, it will be repeated in every custom control creator.  This means you will have to remove defaultControlCreator() as the end of the chain of responsibility, since it doesn't call defineCustom(), but I think that's a good tradeoff.

Reply:
I wasn't sure if this was definitely something every control needed to do. If you're saying it is, then your suggestion sounds reasonable.

Reply:
Oh, and I didn't need to remove the defaultControlCreator() because it returns None, I'm only calling defineCustom(...) if the returned result is not None.